### PR TITLE
Add models for country_groups,country_group_entries,country_mappings

### PR DESCRIPTION
**Pull request by abbas123456**

### CODE REVIEW

File: app/db/models/external.py [hunk 1]
```diff
@@ -231,3 +233,23 @@ class DITBACIL1(BaseModel):
...
+
+
+class LITECountryGroup(BaseModel):
+    __tablename__ = 'country_group'
+
+    id = _col(_int, primary_key=True, autoincrement=True)
+
+
+class LITERefCountryMapping(BaseModel):
+    __tablename__ = 'ref_country_mapping'
```
Comment (marcuspp) [inline app/db/models/external.py:245]: Do you think the table names should be capital letters? Or it doesn't matter?

File: app/db/models/external.py [hunk 2]
```diff
@@ -231,3 +233,23 @@ class DITBACIL1(BaseModel):
...
+
+    country_id = _col(_int, primary_key=True, autoincrement=True)
+    country_name = _col(_text)
+
+
+lite_country_group_entry = _table(
```
Comment (marcuspp) [inline app/db/models/external.py:251]: Can this not be a norma class based model?

Comment (abbas123456) [reply app/db/models/external.py:251]: All the examples I found about many to many did it this way, happy to change it if you can find an class based example

File: app/db/models/external.py (hunk 1 above)
Comment (abbas123456) [reply app/db/models/external.py:245]: The oracle naming convention is all upper case but the postgres convention is all lower case, so I guess would be good to keep it as it is unless you think we should match the LITE/SPIRE data structure

File: app/db/models/external.py [hunk 3]
```diff
@@ -231,3 +233,23 @@ class DITBACIL1(BaseModel):
     importer = _col(_int)
     trade_flow_value = _col(_decimal)
     quantity = _col(_decimal)
+
+
+class LITECountryGroup(BaseModel):
+    __tablename__ = 'country_group'
```
Comment (marcuspp) [inline app/db/models/external.py:239]: Need to set a schema for all the tables to go in.

**Pull request merged**